packages/wallets/src/features/cashier/screens/DepositLocked/DepositLockedProvider.tsx

```diff
@@ -0,0 +1,172 @@
+import React from 'react';
+import { Trans } from 'react-i18next';
+import { WalletLink, WalletText } from '../../../../components';
+
+type TDepositLockedProps = {
+    askFixDetails: boolean;
+    clientTncStatus?: string | null;
+    currency: string;
+    excludedUntil?: Date;
+    financialInformationNotComplete: boolean;
+    isMFAccount: boolean;
+    poaNeedsVerification: boolean;
+    poaStatus: string;
+    poiNeedsVerification: boolean;
+    poiStatus: string;
+    selfExclusion: boolean;
+    tradingExperienceNotComplete: boolean;
+    unwelcomeStatus: boolean;
+    websiteTncVersion?: string;
+};
+
+const depositLockedProvider = ({
```

Comment:
naming - for whatever reason, this makes me think its a part of react context, 
while in fact, its util function / helper function etc, 

better to have something like "getDepositLockInfo" or "isAccountBlocked"

Reply:
Updated naming to getWIthdrawalLockedContent and getDepositLockedContent to make it more understandable 👍